realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,8 +227,57 @@ static void notifyUserLoggedOut(SyncUser user) {
         }
     }
 
+    // This is called from SyncManager.cpp from the worker thread the Sync Client is running on
+    // Right now Core doesn't send these errors to the proper session, so instead we need to notify all sessions
```

Comment:
Some work has done in this regard. We probably need to ask @kspangsege about the current state.